### DISCUSSION TITLE
Default Login

### DISCUSSION THREAD

**Discussion by unixpoin** [Tomiwa-Ot/moukthar#23] [General]
what is default credential for login ? i ca'nt find default credential

**Comment by Tomiwa-Ot** [comment]
It's in the readme.

**Reply by unixpoin** [reply]
i got this : 
![image](https://github.com/Tomiwa-Ot/moukthar/assets/50932471/2e32813c-5ddb-4958-adcc-4489409833a5)

**Comment by unixpoin** [comment]
i wanna try to make tutorial video how to install, but i can't login

**Comment by ohyeah521** [comment]
Deploying with the Python version is straightforward; avoid using the PHP version

**Comment by unixpoin** [comment]
I not a programer bro, i m just trying and any suggest where i can found
python version ?

**Reply by mikelkarma** [reply]
https://github.com/Tomiwa-Ot/moukthar/tree/v1

**Comment by Tomiwa-Ot** [comment]
Youtube tutorial: https://youtu.be/ykOx19hAaD4?feature=shared

**Comment by Ryzen303** [comment]
i try follow tutorial but get error 500 when login

**Reply by Tomiwa-Ot** [reply]
Share the output of this command 
```sh
sudo cat /var/log/apache2/error.log
```
